### PR TITLE
Add match_about_blank to fix #3

### LINKED ISSUE #3: Stylish does not affect dynamically generated iframes
From https://github.com/JasonBarnabe/stylish/issues/103

http://forum.userstyles.org/discussion/15032/how-to-apply-css-to-content-within-an-iframe
https://code.google.com/p/chromium/issues/detail?id=76429

### PR DESCRIPTION
**Pull request by silverwind**
This fixes #3. I've tested it on a problematic `<iframe>` on irccloud.com.

### CODE REVIEW

**Comment by silverwind** [comment]
Also, I'd :heart: you if you could push an update to the Chrome store with this.

File: manifest.json [hunk 1]
```diff
@@ -20,12 +20,14 @@
...
 		{
 			"matches": ["http://*/*", "https://*/*"],
+			"match_about_blank": true,
 			"run_at": "document_start",
 			"all_frames": true,
 			"js": ["apply.js"]
 		},
 		{
 			"matches": ["http://userstyles.org/*", "https://userstyles.org/*"],
+			"match_about_blank": true,
```
Comment (JasonBarnabe) [inline manifest.json]: This block is to handle installs from userstyles.org. It shouldn't have `match_about_blank`.

Comment (silverwind) [inline manifest.json]: Alright, will update.

**Comment by silverwind** [comment]
Updated the PR to not apply to userstyles.org

**Pull request merged**